Open_Science_New.html

```diff
@@ -474,134 +516,24 @@ <h2><font color="0f4078">Real-Time Query Results</font></h2>
             </section>
-	    
-	    <!-- CCNA  -->
-	    <section data-transition="concave">
-	      <table class="reveal">
-		<h2>CCNA</h2>
-		<tr>
-		  <td width="70%" style="vertical-align: top">
-		    <img src="images/CCNA.png">
-		  </td>
-		  <td width="30%" style="vertical-align: top">
-		    <p><small>
-			<ul>
-			  <li>$35 million</li><br>
-			  <li>CIHR funded</li><br>
-			  <li>Pan Canadian Consortium</li><br>
-			  <li>Acquisition in June, 2016</li><br>
-			  <li>35 collections sites</li><br>
-			  <li>LORIS powered</li><br>
-			</ul>
-		    </small></p>
-		</td>
-	      </tr></table>
-	    </section>
-
-	    <!-- Phantom -->
-	  <section>
-	    <img src="images/Phantom.png">
-	  </section>
-	    
-	    <!-- QPN -->
-	    <section data-transition="fade">
-	      <img src="images/QPN.png" width="110%">
-	    </section>
-	    
-	    <!-- PreventAD -->
-	  <section>
-	    <img src="images/PreventAD.png" width="87%">
-	  </section>
 
-	  <!-- Cuban Brain Mapping -->
-	  <section data-transition="convex">
-	    <a href="" target="_blank"><img src="images/Cuban_brain_mapping.png" width="120%"></a>
-	  </section>
-	    <!-- ABIDE -->
-	    <section data-transition="fade">
-	      <table class="reveal">
-		<h2>ABIDE</h2>
+        <!-- Brainhack -->
+	  <section data-transition="fade">
+	    <table class="reveal">
+	      <h2>Hackathons</h2>
 	      <tr>
-		<td width="45%" style="vertical-align: middle">
-		  <img src="images/ABIDE_paper.png">
+		<td width="75%" style="vertical-align: middle">
+		  <img src="images/C-BASE.png">
 		</td>
-		<td width="55%" style="vertical-align: middle">
-		  <img src="images/ABIDE_preprocessed.png">
+		<td width="25%" style="vertical-align: middle">
+		  <p> Important value of hackathons to data sharing initiatives</p>
+		  <a href="http://www.brainhack.org"><img src="images/Brainhack.png"></a>
 		</td>
 	    </tr></table>
-	    </section>
-	  
 	  </section>
-	  
-	
-	
-	<!-- AI -->
-	<section>
-	  <img src="images/DELL_AI.png" width="130%">
-	</section>
-	
-	<!-- Acknowledgemnts-->
+
+<!-- Acknowledgemnts-->
 	<section data-transition="slide" data-background="#87CEEB" data-background-transition="zoom">
 	  <table class="reveal">
 	    <tr>
```

Comment:
- [ ] update the team grid with the 2018 version here: 
http://lorisdb.github.io/about.html

Reply:
update: we'll have a new version up on loris.ca later this week...